rsconf/package_data/mongod/logrotate.conf.jinja

```diff
@@ -2,7 +2,7 @@
     missingok
     notifempty
     postrotate
-        kill -SIGUSR1 $(cat {{ systemd.pid_file }}) &> /dev/null
```

Comment:
This is non-traditional. /bin/kill supports `-USR1` which is how I've always seen it.

Reply:
thanks, c86507f

The `|| true` doesn't strike me as very fail fast. It would've masked the current error we're having right?

Reply:
Yes, it isn't fail fast but we probably don't want to cause a failure in this case. I'm not sure if logrotate continues after errors. Better to buy into the way everybody else does it for this particular problem. I wouldn't write logrotate this way at all, but it's the way it works...